app/partials/coinify/summary.pug

```diff
@@ -1,7 +1,8 @@
 .flex-column.mbl(ng-show="error || $parent.error")
   p.mvm.state-danger-text.type-md(translate="UNABLE_TO_PROCESS")
   span.mvm(ng-hide="isSell" translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.error.error_description}")
-  span.mvm(ng-show="isSell" translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.error.error_description}")
+  span.mvm(ng-show="isSell && !$parent.quoteError" translate="CANNOT_COMPLETE_TRADE_ERROR" translate-values="{error: $parent.error.error_description}")
+  span.mvm(ng-show="isSell && $parent.quoteError" translate="SELL.QUOTE_EXPIRED_ERROR" translate-values="{error: $parent.error.error_description}")
```

Comment:
The `ng-show` check is for `quoteError` but the translation values display `error`, intentional?